pages/your_tasks/metadata_management.md

```diff
@@ -70,19 +70,20 @@ Due to the large variety of experiments, techniques and collaborative studies th
 
 * The established data organization system has to be described in detail in the documentation, preferably in open and machine-readable formats (i.e., XML, JSON, CSV, RDF, HTML). The description of the data organization system has to be placed in the folder at the highest level (e.g. “Project” folder).
 
-* [Study-level and data-level documentation](https://www.ukdataservice.ac.uk/manage-data/document/data-level.aspx) can be provided as
+* [Study-level](https://ukdataservice.ac.uk/learning-hub/research-data-management/document-your-data/study-level-documentation/) and [data-level](https://ukdataservice.ac.uk/learning-hub/research-data-management/document-your-data/data-level/) documentation can be provided as
   * README file
   * [Codebook](https://ddialliance.org/training/getting-started-new-content/create-a-codebook)
   * Data dictionary ([see an example](https://webdav-r3lab.uni.lu/public/elixir/templates/Data_dictionary_example.xlsx))
   * Data list
 
-  Each of these files can be made in several formats depending on the features available in your data documentation tool, your needs or skills. Machine-readable or -actionable formats (such as .xml, .json, .csv, .rdf) are preferred to non-machine-readable ones (.txt, .xls, .pdf). Also non-proprietary formats are preferred over proprietary ones.
+  Each of these files can be made in several formats depending on the features available in your data documentation tool, your needs or skills. Machine-readable or -actionable formats (such as .xml, .json, .csv, .rdf) are preferred to non-machine-readable ones (.txt, .xls, .pdf).  README.txt is an exception from this approach as its main purpose is to be human-readable which is controversial with machine-actionable.
```

Comment:
What is meant by "which is controversial with machine-actionable"? 

"which is contradictory to machine-actionability" ?

Reply:
Text has been changed and improved